CMakeLists.txt

```diff
@@ -1,9 +1,18 @@
 cmake_minimum_required(VERSION 3.3)
 
-set(VERSION_MAJOR 0)
-set(VERSION_MINOR 80)
+set(VERSION_MAJOR 1)
+set(VERSION_MINOR 0)
```

Comment:
Version switch?

Reply:
Nah, it's just that GitHub is broken since I merged master.